### ISSUE TITLE
Ant-Man and the Wasp come before Infinity War

### ISSUE THREAD

**Issue by dareenzo** [dispossible/MCU-Timeline#1]
Using the watch order Ant-Man and the Wasp show before Infinity War, but based on the dates it should be the other way around.

![Screenshot from 2019-04-14 20-03-13](https://user-images.githubusercontent.com/6130991/56097099-70d9e180-5ef0-11e9-96a5-25861999570a.png)

**Comment by dispossible** [comment]
Hey Dareenzo,

Thanks for flagging your concern.

Based on the release dates you are correct they would be the other way around, however as you said you are in "Watch Order" mode, which means the films will show up in the order that I believe is most logical / timeline consistent to make sense when watching through.

The main plot of Ant-Man 2 (AM2) takes place prior to the events of Infinity War (IW), with only the after credits scene taking place during IW. So due to this I decided it was more logical to place AM2 before IW in the watch order.

This was a choice I debated over for some time as it does leave the AM2 post credits scene as something of a spoiler, but possibly also a teaser for what is to come...?

I will leave this issue open and label it as a question, if anyone else feels strongly that this is the incorrect order then I will revisit my decision.
